app/controllers/site_controller.rb

```diff
@@ -56,34 +56,51 @@ def load_global
     @sdgs = YAML.load(File.open("#{Rails.root}/lib/data/content/sdgs.yml", 'r'))
   end
 
+  def sum_country_areas(total_area_by_country)
+    country_total_area = {}
+    total_area_by_country.flatten.each do |country_data|
+      next if country_data["iso3"].include? "/" #remove areas which have multiple iso
+      next if country_data["iso3"].include? "ABNJ" #remove ABNJ
+      country_total_area[country_data["iso3"]] = country_total_area[country_data["iso3"]].nil? ?  country_data["sum"] : country_total_area[country_data["iso3"]] + country_data["sum"]
+    end
+    country_total_area
+  end
+
+  def sort_country_count(total_value_by_country)
+    country_total_points = {}
+    total_value_by_country.each do |total_value|
+      next if total_value["iso3"].include? "ABNJ"
+      country_total_points[total_value["iso3"]] = total_value["count"]
+    end
+    country_total_points
+  end
+
+
   def load_charts_data
-    @chart_greatest_coverage = [
-      {
-        label: 'Australia',
-        value: '94',
-        percent: '94',
-      },
-      {
-        label: 'United Kingdom',
-        value: '63',
-        percent: '63',
-      },
-      {
-        label: 'Spain',
-        value: '75',
-        percent: '75',
-      },
-      {
-        label: 'Italy',
-        value: '50',
-        percent: '50',
-      },
+    c = Carto.new(@habitat.name)
+    total_value_by_country = 0
+    top_five_countries = []
+    arbitrary_value = 0
+
+    if @habitat.name == "coldcorals"
+      total_value_by_country = c.total_points_by_country
+      total_value_by_country = sort_country_count(total_value_by_country)
+    else
+      total_value_by_country = c.total_area_by_country
+      total_value_by_country = sum_country_areas(total_value_by_country)
+    end
+
+    top_five_countries = total_value_by_country.sort_by {|_key, value| value}.last(5)
+    arbitrary_value = top_five_countries.last.last.to_f * 1.05
```

Comment:
Could you please explain this multiplication? Not sure why multiplying by 1.05 🤔

Reply:
I spoke with Roger and we discussed that the arbitrary value would be 5% greater than the highest value. Unless I have misunderstood this?

Reply:
Great, thanks both for claryfing!